xml/2018-10-14.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<email title="Weekly Digest">
+
+  <image
+    width="80"
+    src="https://i.imgur.com/5rJTZsq.jpg"
+    alt="Photo of digestive biscuits on a plate"
+    title=""/>
+  <hr/>
+
+  <text>
+    Hi again! It's time for another digest.
+
+    In the past week, we held four events: the Mums and Dads meeting, Linux talk,
+    Sponsors' Exhibition and the Bloomberg Decentralised App Development workshop.
+    These events were an overwhelming success with a total attendee count over all
+    four events of 1000. Thanks to everyone who participated and a special thanks
+    to all of our volunteers!
+
+    If you have any feedback about our events, feel free to [get in touch](mailto:docsoc@imperial.ac.uk).
+    Additionally, if you want to help out at our events, be sure to join our
+    [DoCSoc: Operations](https://www.facebook.com/groups/1509850436007222/) Facebook group.
```

Comment:
All of our industrial events are catered (usually with pizza). If you have any dietary requirements and if you are planning on attending an event, please make sure you (get in touch)[mailto..] at least two days before the event to tell us. You can use the same email address to give us any feedback about our events as well!
Additionally, if you...